package.json

```diff
@@ -1,23 +1,58 @@
 {
-    "name": "html-flip-book",
-    "version": "1.0.0",
-    "description": "Flipbook component for HTML",
-    "scripts": {},
-    "author": "DoradSoft",
-    "license": "MIT",
-    "devDependencies": {
-        "css-loader": "^7.1.1",
-        "eslint": "^9.3.0",
```

Comment:
github action is not working with 9+ yet, but dependabot will update when available surely

Reply:
I might miss something, why would GitHub action care which eslint version is used?
Nonetheless here is an example of other repository using such version and running lint succesfully: https://github.com/eslint/eslint-plugin-markdown/actions/runs/8912573400/job/24476285524

Reply:
actions is not stated as a dependency, that is why, we can purposefully delete packages from the dependency list, but that might be dangerous